### PR TITLE
minor typo fixes and grammatical changes to setup.md

### PR DESCRIPTION
**Pull request by raynamharris**
This PR fixes a few barely noticeable typos and suggests a minor rewording of a few sentences. I have no strong feelings about wether we keep my suggested rewording or revert to the original, but it did help with my translation effort.

### CODE REVIEW

File: setup.md [hunk 1]
```diff
@@ -69,7 +70,8 @@ conda install -c conda-forge plotnine
 
 ### Miniconda installation
 
-Miniconda is a "light" version of Anaconda. If you install and use Miniconda
+Miniconda is a "light" version of Anaconda.
```
Comment (maxim-belkin) [inline setup.md]: light -> lightweight

**Comment by raynamharris** [comment]
Excellent! Thanks @maxim-belkin! I've made the request change.

File: setup.md [hunk 2]
```diff
@@ -80,35 +82,35 @@ Python 3.x.
...
+After installing Python and the required packages,
+launch a Jupyter Notebook by typing this command in the terminal:
 
 ~~~
 jupyter notebook
 ~~~
 {: .language-bash}
 
-The notebook should open automatically in your browser. If it does not or you
+A Jupyter Notebook notebook should open automatically in your browser. If it does not, or you
```
[the remark this answers is not in the document]
Comment (orchid00) [reply setup.md]: that second notebook is redundant

[the remark this answers is not in the document]
Comment (wrightaprilm) [reply setup.md]:
```suggestion
A Jupyter Notebook should open automatically in your browser. If it does not, or you
```
Removing redundant word. Once you accept this change, I think we can merge!

**Comment by tobyhodges** [comment]
I'm helping the current lesson Maintainers process outstanding pull requests on this repository, in preparation for [transition to the new lesson infrastructure](https://carpentries.github.io/workbench/faq.html#update-fork-from-styles).

The setup page has changed considerably since this PR was opened, and most of the suggested changes have been incorporated elsewhere, or otherwise rendered obsolete by content reorganisation etc. I am going to close this, but thank you very much for contributing @raynamharris

**Pull request closed without merging**